### PR TITLE
laravel 9 fix

### PR DESCRIPTION
**Pull request by belivey**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by belivey** [comment]
l9

File: composer.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+    "name": "belivey/botman",
     "license": "MIT",
     "description": "Create messaging bots in PHP with ease.",
     "keywords": [
         "Bot",
         "BotMan",
         "Chatbot"
     ],
-    "homepage": "http://github.com/botman/botman",
+    "homepage": "http://github.com/belivey/botman",
```
Comment (tinkeshwar) [inline composer.json:10]: change back to botman

File: composer.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
-    "name": "botman/botman",
+    "name": "belivey/botman",
```
Comment (tinkeshwar) [inline composer.json:2]: change back to botmat

**Review by tinkeshwar — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (alies-dev) [inline composer.json:2]:
```suggestion
    "name": "botman/botman",
```

File: composer.json (hunk 1 above)
Comment (alies-dev) [inline composer.json:10]:
```suggestion
    "homepage": "http://github.com/botman/botman",
```

**Review by alies-dev — COMMENTED** [review]
let's finally merge this PR

**Pull request closed without merging**